src/fermilib/utils/_operator_utils.py

```diff
@@ -223,3 +217,19 @@ def get_file_path(file_name, data_directory):
         file_path = data_directory + '/' + file_name
 
     return file_path
+
+
+def numpy_scalar(number):
```

Comment:
What if we were to cast to complex all float types to complex instead? This makes me wonder why we don't just make the coefficients complex when they're defined - is there a reason we need things like numpy.complex128? @damiansteiger @babbush @jarrodmcc